content/en/docs/concepts/storage/volume-snapshots.md

```diff
@@ -27,108 +29,108 @@ A `VolumeSnapshotContent` is a snapshot taken from a volume in the cluster that
+There are two ways snapshots may be provisioned: pre-provisioned or dynamically provisioned.
 
-#### Static
-A cluster administrator creates a number of `VolumeSnapshotContents`. They carry the details of the real storage which is available for use by cluster users. They exist in the Kubernetes API and are available for consumption.
+#### Pre-provisioned {#static}
+A cluster administrator creates a number of `VolumeSnapshotContents`. They carry the details of the real volume snapshot on the storage system which is available for use by cluster users. They exist in the Kubernetes API and are available for consumption.
 
 #### Dynamic
-When none of the static `VolumeSnapshotContents` the administrator created matches a user's `VolumeSnapshot`,
-the cluster may try to dynamically provision a volume snapshot specially for the `VolumeSnapshot` object.
-This provisioning is based on `VolumeSnapshotClasses`: the `VolumeSnapshot` must request a
-[volume snapshot class](/docs/concepts/storage/volume-snapshot-classes/) and
-the administrator must have created and configured that class in order for dynamic
-provisioning to occur.
+Instead of a pre-existing snapshot, a user can request that a snapshot of a PVC is dynamically taken. The [VolumeSnapshotClass](/docs/concepts/storage/volume-snapshot-classes/) specifies storage provider-specific parameters to use when taking a snapshot.
 
 ### Binding
 
-A user creates, or has already created in the case of dynamic provisioning, a `VolumeSnapshot` with a specific amount of storage requested and with certain access modes. A control loop watches for new VolumeSnapshots, finds a matching VolumeSnapshotContent (if possible), and binds them together. If a VolumeSnapshotContent was dynamically provisioned for a new VolumeSnapshot, the loop will always bind that VolumeSnapshotContent to the VolumeSnapshot. Once bound, `VolumeSnapshot` binds are exclusive, regardless of how they were bound. A VolumeSnapshot to VolumeSnapshotContent binding is a one-to-one mapping.
+The snapshot controller handles the binding of a `VolumeSnapshot` object with an appropriate `VolumeSnapshotContent` object, in both pre-provisioned and dynamically provisioned scenarios. The binding is a one-to-one mapping.
 
 VolumeSnapshots will remain unbound indefinitely if a matching VolumeSnapshotContent does not exist. VolumeSnapshots will be bound as matching VolumeSnapshotContents become available.
 
-### Persistent Volume Claim in Use Protection
+### Persistent Volume Claim as Snapshot Source Protection
 
-The purpose of the Persistent Volume Claim Object in Use Protection feature is to ensure that in-use PVC API objects are not removed from the system (as this may result in data loss).
+The purpose of the Persistent Volume Claim Object while a snapshot is being taken Protection feature is to ensure that in-use PVC API objects are not removed from the system (as this may result in data loss).
```

Comment:
> The purpose of the Persistent Volume Claim Object while a snapshot is being taken Protection feature is to ensure that in-use PVC API objects are not removed from the system (as this may result in data loss).
Ouch, that's hard to parse.

How about:
```suggestion
Kubernetes protects PersistentVolumeClaim objects during snapshotting operations.
```

(if that protection isn't done using a built-in controller, then replace “Kubernetes” with a description of what implements the protection).

Reply:
I shortened the sentence a little bit.  Hope that reads better.